demos/lane_detection_demo/python/lane_detection_demo.py

```diff
@@ -0,0 +1,330 @@
+from __future__ import print_function, division
```

Comment:
do you really need these? Python2 reached end of life status in 2020, minimal supported python version for openvino - 3.6